books/management/commands/addepub.py

```diff
@@ -39,8 +39,6 @@ def validate_and_add(path, filenames):
         if os.path.splitext(path)[1] == '.epub':
             filename = os.path.abspath(path)
             if filename not in filenames:
-                if not models.Book.objects.filter(
-                        original_path=filename).exists():
```

Comment:
Gotcha! And yeah, I fully agree it probably provides quite a speedup in some scenarios (importing a folder that contains just a few new epubs, etc).

As a matter of fact I will add a commit to this pull request in order to make that check optional (and probably defaulting to True), to cover another set of scenarios (the user has moved his source files around or renamed them, and for some reason wants to be sure that all files are taken into account, even if it means a slower import).